libs/utils/src/hooks/useFetchOmdb.tsx

```diff
@@ -0,0 +1,30 @@
+import { Film } from '@form-exercise/data/interface';
+import axios, { AxiosError } from 'axios';
+import { useQuery } from 'react-query';
+
+interface UseFetchOmdb {
+  searchBy: string;
+  type?: string;
+}
+
+interface Films {
+  Reponse: string;
+  Search: Film[];
+  totalResults: string;
+}
+
+export function useFetchOmdb({ searchBy, type = 'movie' }: UseFetchOmdb) {
+  const getFilms = async (): Promise<Films> => {
```

Comment:
create a hook by action, if this api has a dozen endpoint, this hook will be too bloated, see previous comment